env0/resource_environment.go

```diff
@@ -418,7 +424,7 @@ func shouldDeploy(d *schema.ResourceData) bool {
 }
 
 func shouldUpdate(d *schema.ResourceData) bool {
-	return d.HasChanges("name", "approve_plan_automatically", "deploy_on_push", "run_plan_on_pull_requests", "auto_deploy_by_custom_glob", "auto_deploy_on_path_changes_only", "terragrunt_working_directory", "vcs_commands_alias")
```

Comment:
IIRC you can't update `isRemoteBackend` for an existing environment, no?

This seems like a bit of a unique case. We'd want to prevent terraform from allowing a user to update this field

Reply:
OK nvm, we added that to the `update` lambda, apparently